moveit_py/test/unit/fixtures/panda.srdf

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I agree this would be ideal, I would need to create a GitHub action to keep this file synced with changes from https://github.com/ros-planning/moveit_resources/tree/ros2.

Reply:
https://github.com/peterdavidfagan/moveit2/blob/moveit_py/moveit_core/utils/src/robot_model_test_utils.cpp

Reply:
[642680e](https://github.com/ros-planning/moveit2/pull/1546/commits/642680ec7fb9d16e470d314b1a50201b1b967b65)